build/version.props

```diff
@@ -1,17 +1,22 @@
 <!-- This file may be overwritten by automation. Only values allowed here are VersionPrefix and VersionSuffix.  -->
 <Project>
```

Comment:
Can we delete this file?

Reply:
good catch, I had meant to delete it. it's deleted now.